docs/serving/config-ha.md

```diff
@@ -10,7 +10,7 @@ Active/passive HA in Knative is available through leader election, which can be
 
 When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
 
-HA functionality is enabled by default for all Knative Serving components except for the `autoscaler` component.
+Leader election is enabled but HA functionanlity is disabled with 1 replica by default for all Knative Serving components.
```

Comment:
```suggestion
Leader election is enabled by default for all Knative Serving components.
HA functionality is disabled by default for all Knative Serving components, which are configured with only one replica.
```
I feel like we should have a note about enabling HA though by changing the replica config or point to docs about it here?

Reply:
hmm, like `See Scaling the control plane section for how to enable HA`?